plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx

```diff
@@ -45,7 +51,10 @@ export interface RepoUrlPickerUiOptions {
 }
 
 /**
- * Repo Url Picker
+ * The underling component that is rendered in the form for the `RepoUrlPicker`
```

Comment:
```suggestion
 * The underlying component that is rendered in the form for the `RepoUrlPicker`
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>